connectors/vault/go.mod

```diff
@@ -7,7 +7,17 @@ replace github.com/ibm/the-mesh-for-data v0.0.0 => ../..
 go 1.13
 
 require (
+	//https://github.com/kubernetes/client-go/issues/628
```

Comment:
remove this file (not needed once building outside docker)

Reply:
fixed. Thanks.

Reply:
In this PR we need to edit the private go.mod to have some packages exclusively for running unit tests. In we do not have a private go.mod then we have to move the contents of this file to the root level go.mod. If we do that, when we run make verify, the command automatically removes all packages not used in go files causing the unit tests to fail. The make verify command does not seem to check if the package is being used in test files. Hence for this purpose, one option is to keep using the private go.mod file with specific packages needed for test files listed in this private go.mod file. Please suggest if there are any other alternatives to this issue. Thanks.